ControlRoom/Controllers/SimulatorsController.swift

```diff
@@ -130,8 +130,19 @@ class SimulatorsController: ObservableObject {
         guard loadingStatus == .success else { return }
 
         let trimmed = preferences.filterText.trimmingCharacters(in: .whitespacesAndNewlines)
-
         var filtered = allSimulators
+
+        if preferences.showBootedDevicesFirst {
+            let on = filtered.filter { $0.state != .shutdown }
+            let off = filtered.filter { $0.state == .shutdown }
+            filtered = on.sorted() + off.sorted()
+        } else {
+            filtered = filtered.sorted()
+        }
```

Comment:
Would it make sense to make the default be sorted by state then by name. This would require updating the comparable operator in `Simulators.swift` but then the condition could be a lot simpler

```
switch preferences.showBootedDevicesFirst {
case true: filtered.sorted()
case false: filtered.sorted { $0.name < $1.name }
}
```

Reply:
@joeblau - perhaps... That should be a separate issue though.

Reply:
Yeah true